content/docs/user-guide/packaging/windows-release-builds.md

```diff
@@ -0,0 +1,296 @@
+
+
+#### Create a new seed asset list
+Create a new see asset list that contains only your level assets. 
+
+2. Create a new seed list file by clicking **Create a new Seed List file** in the **Seed List file** panel under the **Seeds** tab
+3. Select your new seed list file from the list. 
+4. Click **+ Add Asset** in the **Product Assets** panel. This opens the **Add Seed Asset** dialog. 
+5. In the list of platforms, enable **pc**.
+6. Click **Browse...** and navigate to the `levels` folder and choose **Add Seed**. 
+7. Verify that your seed list has the level assets in the **Product Assets** panel. 
+
+#### Generate an asset list 
+
+1. Select your new seed list file by enabling the check box. 
+2. Click **Generate Asset Lists**. This will open the **Generate Asset List files** dialog. 
+3. In the list of platforms, select **pc** by enabling the check box.  
+4. Click **Browse...** to open the File Explorer. 
+5. Enter a name for your asset list and save. In this example, we'll use the name `game.assetlist`. 
+5. Navigate to the **Asset Lists** tab to verify the assets in `game.assetlist`. The assets are listed under the **Asset List** panel. 
+
+#### Bundle your assets
+
+1. Select your `game.assetlist` in the** Asset List Files** panel under the **Asset Lists** tab.
+
+2. Click **Generate Bundle**. This opens the **Generate Bundles** dialog. 
+
+3. Click **Browse...** to open the File Explorer. 
+
+4. Enter a name for your package file and save. In this example, we'll use the name `game_bundle.pak`.
+
+
+### Create a bundle for engine assets
+
+Now, you'll create a bundle for your project's engine assets. 
+
+#### Generate an asset list from default seed lists
+
+1. Navigate to the Seeds tab. 
+
+2. Select **Default Seed Lists** at the bottom of the **Seed List files** panel. Make sure to deselect other seed list files. 
+
+2. Click **Generate Asset Lists**. This will open the **Generate Asset List files** dialog. 
+
+3. In the list of platforms, select **pc** by enabling the check box.  
+
+4. Click **Browse...** to open the File Explorer. 
+
+5. Enter a name for your asset list and save. In this example, we'll use the name `engine.assetlist`. 
+
+5. Navigate to the **Asset Lists** tab to verify the assets in `engine.assetlist`. The assets are listed under the **Asset List** panel. 
+
+#### Bundle your assets
+
+1. Select your `engine.assetlist` in the** Asset List Files** panel under the **Asset Lists** tab.
+
+2. Click **Generate Bundle**. This opens the **Generate Bundles** dialog. 
+
+3. Click **Browse...** to open the File Explorer. 
+
+4. Enter a name for your package file and save. In this example, we'll use the name `engine_bundle.pak`.
+
+
+You now have two bundled content: `game_bundle.pak` and `engine_bundle.pak`. 
+
+
+### Add your bundles to install layout
+
+Next, add `game_bundle.pak` and `engine_bundle.pak` to `install/bin/Windows/release/Default`. You can remove the `engine.pak` file that was created earlier. 
```

Comment:
```suggestion
Next, add your `game_bundle.pak` and `engine_bundle.pak` files to `install/bin/Windows/release/Default`. You can remove the `engine.pak` file created earlier.
```
* Minor edits for flow.
* Is the action in the second sentence ("you can remove the `engine.pak` file") optional or required? Clarify by adding "Optionally, you can..." or "Be sure to remove...".